abis_mapping/templates/survey_site_visit_data_v2/schema.json

```diff
@@ -19,8 +19,7 @@
 			"type": "string",
 			"format": "default",
 			"constraints": {
-				"required": true,
-				"unique": true
+				"required": true
```

Comment:
@joecrowleygaia do you agree with removing `unique: true` from `siteID`? Otherwise you won't be able to have multiple visits per site in the template